utils/utils.go

```diff
@@ -67,9 +67,14 @@ func RunUnderSystemdScope(pid int, slice, unitName string, properties ...systemd
 		properties = append(properties, systemdDbus.PropSlice(slice))
 	}
 	ch := make(chan string)
-	_, err = conn.StartTransientUnit(unitName, "replace", properties, ch)
-	if err != nil {
-		return err
+	for {
+		_, err = conn.StartTransientUnit(unitName, "replace", properties, ch)
+		if err == nil {
+			break
+		}
+		if !errors.Is(err, syscall.EAGAIN) {
```

Comment:
Hmm, interesting... @haircommander  where the need for retry-on-EAGAIN comes from?

Asking because maybe we need to handle the same error in runc.

Reply:
OK, this comes from #4921